<?xml version="1.0"?>
<launch>
    <rosparam command="load" file="$(find color_detector_params)/config/config.yaml" ns="color_detector_params" />
    <node pkg="point_cloud_color_detector" type="point_cloud_color_detector" name="point_cloud_color_detector" output="screen" />
</launch>